client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -40,7 +40,7 @@ class SettingsPage extends Component {
 		const content = isConnected
 			? translate( 'One-click restores are enabled.' )
 			: translate(
-					'Enter your server credentials to enable one-click restores for Backups. {{a}}Find your server credentials{{/a}}',
+					'Enter your server credentials to enable one-click restores for Backups. {{a}}Need help? Find your server credentials{{/a}}',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.